DuckDuckGoTests/APIRequestTests.swift

```diff
@@ -117,6 +117,30 @@ class APIRequestTests: XCTestCase {
         }
         waitForExpectations(timeout: 1.0, handler: nil)
     }
+    
+    func testWhenMultipleRequestsAreFiredThenCallbacksAreProcessedOnSerialQueue() {
+        
+        stub(condition: isHost(host)) { _ in
+            return fixture(filePath: self.validJson(), status: 200, headers: nil)
+        }
+        
+        let expectFirst = expectation(description: "first request has been processed")
+        let expectLast = expectation(description: "second request has been processed")
+        
+        APIRequest.request(url: url) { (_, error) in
+            // Give second request a chance to execute the callback
```

Comment:
I think this comment would be more useful if it also said why. Would it be correct if we added "because it shouldn't since the requests are run serially." ?

Reply:
Actually, the test name does imply that so maybe that's unnecessary. Your call.

Reply:
I'd keep it as it is now. I think - like you mentioned - that the test name clarifies this.